backend/endpoints/v2/index.ts

```diff
@@ -37,11 +38,28 @@ const Searches = Endpoint({
   Output: t.any,
 });
 
+const AddEvents = Endpoint({
+  Method: "POST",
+  getPath: () => `/v2/events`,
+  Input: {
+    Headers: t.type({
+      "X-YTtrex-Version": t.string,
+      "X-YTtrex-Build": t.string,
+      "X-YTtrex-NonAuthCookieId": t.string,
```

Comment:
the cookie id is something we're not reading anymore! it is empty, and it can be safely be removed

Reply:
aand removed!